conf/mesos-site.xml

```diff
@@ -19,54 +19,84 @@
 <configuration>
   <property>
     <name>mesos.hdfs.data.dir</name>
+    <description>The primary data directory in HDFS</description>
     <value>/tmp/hdfs/data</value>
   </property>
   
   <property>
     <name>mesos.hdfs.secondary.data.dir</name>
+    <description>The secondary data directory in HDFS</description>
     <value>/var/run/hadoop-hdfs</value>
   </property>
-  
+
+  <property>
+    <name>mesos.hdfs.native-hadoop-binaries</name>
+    <description>Mark true if you have hadoop pre-installed on your host machines (otherwise it will be distributed by the scheduler)</description>
+    <value>false</value>
+  </property>
+
   <property>
     <name>mesos.hdfs.framework.mnt.path</name>
+    <description>Mount location (if above config is marked false)</description>
     <value>/opt/mesosphere</value>
   </property>
 
   <property>
     <name>mesos.hdfs.state.zk</name>
     <value>localhost:2181</value>
+    <description>Hostname and port of zookeeper node location</description>
   </property>
 
   <property>
     <name>mesos.master.uri</name>
     <value>zk://localhost:2181/mesos</value>
+    <description>Zookeeper entry for mesos master location</description>
   </property>
 
   <property>
     <name>mesos.hdfs.zkfc.ha.zookeeper.quorum</name>
     <value>localhost:2181</value>
+    <description>Comma-separated list of zookeeper hostname-port pairs</description>
   </property>
 
   <property>
     <name>mesos.hdfs.framework.name</name>
+    <description>Your cluster name when accessing files (hdfs://YOUR_NAME)</description>
     <value>hdfs</value>
   </property>
 
   <property>
     <name>mesos.hdfs.mesosdns</name>
+    <description>Whether to use Mesos DNS for service discovery within HDFS</description>
     <value>false</value>
   </property>
 
   <property>
     <name>mesos.hdfs.mesosdns.domain</name>
+    <description>Root domain name of Mesos DNS (usually 'Mesos')</description>
```

Comment:
Also, this can be ignored/commented-out if mesos.hdfs.mesosdns defaults to false.